Makefile

```diff
@@ -661,21 +661,18 @@ test-e2e: ## Run "docker-build" and "docker-push" rules then run e2e tests.
 	$(MAKE) docker-build docker-push \
 	test-e2e-run
 
-LOCAL_GINKGO_ARGS ?= -stream --progress
-LOCAL_GINKGO_ARGS += $(GINKGO_ARGS)
 .PHONY: test-e2e-local
 test-e2e-local: ## Run "docker-build" rule then run e2e tests.
 	PULL_POLICY=IfNotPresent MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG) \
 	$(MAKE) docker-build \
-	GINKGO_ARGS='$(LOCAL_GINKGO_ARGS)' \
```

Comment:
Just wondering, is this no longer needed in Ginkgo v2?

Reply:
@Jont828 that was removed based on this https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2773#issuecomment-1329727450 above